make/common/templates/adminserver/env

```diff
@@ -11,6 +11,7 @@ LDAP_FILTER=$ldap_filter
 LDAP_UID=$ldap_uid
 LDAP_SCOPE=$ldap_scope
 LDAP_TIMEOUT=$ldap_timeout
+LDAP_VERIFY_CERT=true
```

Comment:
Should read the config from harbor.cfg.

Reply:
This value is not intend to config in harbor.cfg

Reply:
Why isn't it configured in harbor.cfg as other properties are?

Reply:
If it can be re-configured in harbor admin console, it doesn't have to put in harbor.cfg